File: packages/pixlive/android/src/main/java/io/capawesome/capacitorjs/plugins/pixlive/classes/results/GetContextResult.java
package io.capawesome.capacitorjs.plugins.pixlive.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.pixlive.interfaces.Result;

public class GetContextResult implements Result {

    @NonNull
    private final JSObject context;

    public GetContextResult(@NonNull JSObject context) {
        this.context = context;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("context", context);
        return result;
    }
}

File: packages/pixlive/ios/Plugin/Classes/Results/GetContextResult.swift
import Foundation
import Capacitor

@objc public class GetContextResult: NSObject, Result {
    let context: JSObject

    init(context: JSObject) {
        self.context = context
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["context"] = context
        return result as AnyObject
    }
}
